pkg/codeconfig/codeconfig.go

```diff
@@ -311,25 +310,19 @@ func (c *codeConfig) collectOne(handler string) error {
 	if output.VerboseLevel > 2 {
 		pterm.Debug.Println(containerengine.Cli(cc, hostConfig))
 	}
-	logreader, err := ce.ContainerLogs(cID, types.ContainerLogsOptions{
-		ShowStdout: true,
-		ShowStderr: true,
-		Follow:     true,
-	})
-	if err != nil {
-		return err
-	}
-
-	logWriter := log.Writer()
-	logRW := &bytes.Buffer{}
-	if output.VerboseLevel <= 1 {
-		// if we are running in non-verbose then store the container logs in a buffer in case
-		// there are errors.
-		logWriter = logRW
+	if output.VerboseLevel > 1 {
+		logreader, err := ce.ContainerLogs(cID, types.ContainerLogsOptions{
+			ShowStdout: true,
+			ShowStderr: true,
+			Follow:     true,
+		})
+		if err != nil {
+			return err
+		}
+		go func() {
+			_, _ = stdcopy.StdCopy(log.Writer(), log.Writer(), logreader)
```

Comment:
this looks like a rebase error

Reply:
Currently doing a rebase, dropping changes to this file.